### PR TITLE
Allow runtime classing of widgets

### PR DESCRIPTION
**Pull request by GordonSmith**
Signed-off-by: Gordon Smith <gordon.smith@gmail.com>

### CODE REVIEW

**Comment by GordonSmith** [comment]
@mlzummo Please Review

File: src/graph/Edge.js [hunk 1]
```diff
@@ -104,6 +105,7 @@
 
     Edge.prototype.update = function (domNode, element, transitionDuration, skipPushMarkers) {
         SVGWidget.prototype.update.apply(this, arguments);
+        element.classed(this.classed());
```
Comment (mlzummo) [inline src/graph/Edge.js]: whats a proper use example of this publish parameter? you have an object specified but i d3 selection classed doesnt take an obj as far as i know and i cant find the this.classed() function either

**Comment by GordonSmith** [comment]
@jbrundage Please Review

**Comment by mlzummo** [comment]
this is def a good idea ... looks good .... been needing this for a while

**Comment by jbrundage** [comment]
@GordonSmith Looks good to me

**Pull request merged**